src/commands/mathspace/mathspaceCommands.js

```diff
@@ -6,9 +6,23 @@ LatexCmds.simeq = bind(VanillaSymbol, '\\simeq ', '&#11003;');
 
 LatexCmds.interleave = bind(VanillaSymbol, '\\interleave', '&#10996;');
 
-// Map * to times instead of dot
+// MatHSPaCE HacK Map * to times instead of dot
 CharCmds['*'] = LatexCmds.times;
 
+// MatHSPaCE HacK - Allow multiplication sign to be configurable
+Options.p.multiplicationDisplaySymbol = {};
```

Comment:
Does this line need to exist?

My understanding (could be wrong though) is that you would only mutate the Options p(rototype) if you wanted to provide the *default* value for the option.

So it seems weird giving this a default value we'd never actually want it to have. We'd only want it to ever be "cross" | "dot"

Reply:
Oh good point. I was just following the pattern of another option and didn't consider if this made sense. I'll set it to 'cross' which is our actual default